modules/ROOT/pages/rtc/configuration/rtc-options-overview.adoc

```diff
@@ -16,22 +16,22 @@ The Real-time Collaboration (RTC) plugin uses promise-based "provider" functions
 
 These options are _required_ when using the Real-time Collaboration (RTC) plugin.
 
-link:{baseurl}/rtc/configuration/rtc-options-required/#rtc_document_id[`rtc_document_id`]:: Provide a unique identifier for current document (editor content).
+xref:/rtc/configuration/rtc-options-required.adoc#rtc_document_id[`rtc_document_id`]:: Provide a unique identifier for current document (editor content).
```

Comment:
Just a heads up, these xref's shouldn't have the initial `/`, so it should be something like this instead:
```
xref:rtc/configuration/rtc-options-required.adoc#rtc_document_id
```
It does seem to work though, but it's not how we've converted things elsewhere and isn't what's mentioned in the [docs](https://docs.antora.org/antora/latest/page/page-links/#pages) so might only be working by chance.

Reply:
ok, thanks, I'll fix it

Reply:
done